src/uiTest/resources/UITestSuite.conf

```diff
@@ -0,0 +1,24 @@
+probe {
+  driver {
+    vmOptions = ["-Xmx1G"]
+    check.errors = true
+  }
+
+  resolvers {
+    intellij.repositories = [
+      # Let's first try using the IntelliJs downloaded by intellij-plugin-verifier, if present
+      # They're easier to use than Gradle-downloaded IntelliJs since their paths don't contain version-dependent hashes
+      "file:///"${HOME}"/.pluginVerifier/ides/IC-[revision]/",
```

Comment:
That's the new stuff! possible thanks to recent updates in ide-probe implemented by @wiacekm 🙇🏻